website/i18n/en.json

```diff
@@ -388,6 +394,27 @@
       },
       "version-v1.7.0-videos": {
         "title": "Videos"
+      },
```

Comment:
hmm, why were these changes generated?

Reply:
I believe they were an artifact of running `yarn build` and they _seemed_ like they should have been committed previously and so I thought I’d include them. I’m fine with removing them.